modules/gapi/include/opencv2/gapi/gmat.hpp

```diff
@@ -209,7 +209,7 @@ static inline GMatDesc empty_gmat_desc() { return GMatDesc{-1,-1,{-1,-1}}; }
 GAPI_EXPORTS GMatDesc descr_of(const cv::UMat &mat);
 #endif // !defined(GAPI_STANDALONE)
 
-GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat);
+GAPI_EXPORTS GMatDesc descr_of(const cv::Mat &mat); //another way: previous declaration
```

Comment:
And we get error: conflicts with a previous declaration (212 str.).

We include opencv_includes.hpp (cvdefs.hpp in STANDALONE) in gmat.hpp.

cvdefs.hpp: `cv::gapi::own::descr_of(const gapi::own::Mat &)`

gmat.hpp: `cv::descr_of(const gapi::own::Mat &)`

deadlock...